web/models/user.ex

```diff
@@ -6,6 +6,7 @@ defmodule Shareguru.User do
     field :email, :string
     field :picture, :string
     field :expire_at, Ecto.DateTime
+    field :hd, :string, virtual: true # Mail domain
```

Comment:
This data doesn't belong in model, and it's already in the `email` field. I'd rather just rely on the simplest format check: `|> validate_format(:email, ~r/@netguru\.(co|pl)$/)`